cns/cns_util.go

```diff
@@ -51,6 +51,23 @@ func GetTaskResult(ctx context.Context, taskInfo *vim25types.TaskInfo) (cnstypes
 	return nil, errors.New("TaskInfo result is empty")
 }
 
+// GetTaskResultArray gets the task result array for specified a task info
```

Comment:
`GetTaskResultArray gets the task result array for specified a task info` ==>` GetTaskResultArray gets the task result array for a specified task info`

Reply:
fixed